cql3/statements/cas_request.cc

```diff
@@ -101,7 +101,7 @@ lw_shared_ptr<query::read_command> cas_request::read_command() const {
         ranges.reserve(op.ranges.size());
         std::copy(op.ranges.begin(), op.ranges.end(), std::back_inserter(ranges));
     }
-    uint32_t max_rows = query::max_rows;
+    uint64_t max_rows = query::partition_max_rows;
```

Comment:
Is it intentionally changed from max_rows to partition_max_rows?

Reply:
Yes, the variable is later used in set_partition_row_limit so I wanted to use the max_rows per partition.(although both hold the same value)